carsus/io/output/tardis_.py

```diff
@@ -5,18 +5,41 @@
 import re
 
 from pandas import HDFStore
-from sqlalchemy import and_, case
+from sqlalchemy import (
+        and_,
+        case,
+        func,
+        )
 from sqlalchemy.exc import OperationalError
 from sqlalchemy.orm import joinedload, aliased
 from sqlalchemy.orm.exc import NoResultFound
 from astropy import constants as const
 from astropy import units as u
 from scipy import interpolate
 from pyparsing import ParseException
-from carsus.model import Atom, Ion, Line, Level, DataSource, ECollision, MEDIUM_AIR, MEDIUM_VACUUM
+from carsus.model import (
+        Atom,
+        Ion,
+        Line,
+        Level,
+        LevelEnergy,
+        DataSource,
+        ECollision,
+        MEDIUM_AIR,
```

Comment:
yes a docustring - but you don't need to do the full 

```python

Parameters
------------
xxx
```

But just a quick line what this does (for the private functions).